packages/manager-components/src/components/card/card.component.tsx

```diff
@@ -34,35 +34,46 @@ export interface CardProps {
     category: string;
   };
   badges?: Badge[];
-  trackingLabel?: string;
+  dataTrackingPath?: string;
+  dataTrackingValue?: string;
   hoverable?: boolean;
 }
 
-export const Card = ({
+export const Card: React.FC<CardProps> = ({
   href,
   isExternalHref,
   img,
   badges,
   texts,
-  trackingLabel,
+  dataTrackingPath,
+  dataTrackingValue,
   hoverable,
-}: CardProps) => {
+}) => {
   const { title, description, category } = texts;
   const { t } = useDynamicTranslation('card');
+  const { trackClick } = useOvhTracking();
 
   return (
     <a
       target={OdsHTMLAnchorElementTarget._blank}
       rel={OdsHTMLAnchorElementRel.noopener}
       href={href}
+      onClick={() => {
```

Comment:
We are refactoring this part currently so we potentially won't use the core/application package anymore
It's already not used on Rancher, notably because the navigation doesn't work properly